Fixes issue #150 #175

NiteshKant opened the pull request:
#150: Removed all deprecated methods for PoolStats
#175: Provided a method to disable system time calls for metrics.

Commit:
Fixes issue #150

Removed all deprecated methods for PoolStats

cloudbees-pull-request-builder commented:
[RxNetty-pull-requests #105](https://netflixoss.ci.cloudbees.com/job/RxNetty-pull-requests/105/) FAILURE
Looks like there's a problem with this pull request

Commit:
Perf optimization (disable system calls to get time)

In benchmarks where metrics are not required, the system calls to get time (System.currentTimeMillis()) may introduce an unnecessary overhead.
This change enables such benchmarks to disable these system calls to get metric durations.

It is important to note that apart from these system time calls metric generation does not have an object allocation overhead. So, for any benchmarks the metrics infrastructure should not pose much overhead.

cloudbees-pull-request-builder commented:
[RxNetty-pull-requests #106](https://netflixoss.ci.cloudbees.com/job/RxNetty-pull-requests/106/) SUCCESS
This pull request looks good
